### PR TITLE
Optionally pull files from remote host for building ISO files.

### PR DESCRIPTION
**Pull request by dansajner**
This change removes the isolinux files from this repo.

### CODE REVIEW

File: lib/iso.rb [hunk 1]
```diff
@@ -35,17 +40,27 @@ def execute options
...
     tmp_dir = "#{work_dir}/#{hostname}"
-
-    # TODO: handle exceptions
     FileUtils.mkdir_p tmp_dir
 
-    # create the ISO template directory
-    FileUtils.cp_r srcdir, "#{tmp_dir}/isolinux"
+    # Fetch isolinux content.
+    if options[:isolinux_remote_host]
+      remote_path = "/var/satellite/rhn/kickstart/ks-rhel-x86_64-server-#{options[:major]}-#{options[:major]}.#{options[:minor]}/isolinux"
```
Comment (everestx) [inline lib/iso.rb]: Is this a default path for Satellite's storage of boot ISOs?

File: lib/iso.rb [hunk 2]
```diff
@@ -35,17 +40,27 @@ def execute options
...
+
+      require 'net/scp'
+      Net::SCP.start(options[:isolinux_remote_host], ENV['USER']) do |scp|
+        scp.download! remote_path, tmp_dir, options = { :recursive => true }
```
Comment (everestx) [inline lib/iso.rb]: Does this have to be scp or is it available with https?

**Comment by everestx** [comment]
I don't have any strong opinions about the comments above.  Just mentioning them for posterity. We may have to alter the use of scp or the path to the ISOs in the future with satellite 6. 

Looks good.

File: lib/iso.rb (hunk 1 above)
Comment (dansajner) [inline lib/iso.rb]: Yes.

File: lib/iso.rb (hunk 2 above)
Comment (dansajner) [inline lib/iso.rb]: I don't find an api method for accessing these and the are not directly accessible via https.

**Pull request merged**